app/forms/application_form.rb

```diff
@@ -19,16 +19,4 @@ def self.user_editable_attributes
   def self.immutable_attributes
     []
   end
-
-  # @param deposit [Boolean] whether the form is being used for a deposit
-  def valid?(deposit: false)
-    @deposit = deposit
-    super
-  end
```

Comment:
Neither ActiveRecord nor ActiveModel objects support kwargs in `#valid?`, and we want our forms to be Rails-y.